internal/certs/certs.go

```diff
@@ -0,0 +1,153 @@
+package certs
+
+import (
+	"crypto/x509/pkix"
+	"fmt"
+	"github.com/openshift/library-go/pkg/crypto"
+	"github.com/pkg/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"path/filepath"
+	"time"
+)
+
+type CertManager struct {
```

Comment:
```suggestion
type KubeConfigCertManager struct {
```

To avoid giving the impression that this is some generic thing. This is focused solely on kubeconfig